setup.py

```diff
@@ -274,7 +279,7 @@ def setup_package():
             if f[0] != '_'
         ],
         cmdclass=dict(build_ext=CMakeAugmentedBuilder, docs=Docs),
-        install_requires=['numpy>=1.13.1'],
+        install_requires=['numpy>=1.9.3'],
```

Comment:
The version... is the one from test.pypi :) ?

Reply:
yes! I think this is ok because users will get something newer than this anyway.